x509-cert/src/ext/pkix/sct.rs

```diff
@@ -0,0 +1,733 @@
+#![cfg(feature = "sct")]
+//! Signed Certificate Timestamp list extension as defined in the
+//! [Certificate Transparency RFC 6962].
+//!
+//! [Certificate Transparency RFC 6962]: https://datatracker.ietf.org/doc/html/rfc6962
+
+use alloc::{format, vec::Vec};
+use const_oid::{AssociatedOid, ObjectIdentifier};
+use der::asn1::OctetString;
+use tls_codec::{
+    DeserializeBytes, SerializeBytes, TlsByteVecU16, TlsDeserializeBytes, TlsSerializeBytes,
+    TlsSize,
+};
+
+/// A signed certificate timestamp list (SCT list) as defined in [RFC 6962 Section 3.3].
+///
+/// ```text
+/// SignedCertificateTimestampList ::= OCTET STRING
+/// ```
+///
+/// [RFC 6962 Section 3.3]: https://datatracker.ietf.org/doc/html/rfc6962#section-3.3
+#[derive(Debug, PartialEq)]
+pub struct SignedCertificateTimestampList(OctetString);
+
+//TODO: Remove this and use const-oid's rfc6962::CT_PRECERT_SCTS once a const-oid version
```

Comment:
TODO(baloo): make a backport of #1094 and publish a release on 0.9

Reply:
https://github.com/RustCrypto/formats/pull/1282

Reply:
https://docs.rs/const-oid/0.9.6/const_oid/db/rfc6962/constant.CT_PRECERT_SCTS.html

Reply:
Will leave this in for now but we can address it as a followup